assets/locales/en/translation.json

```diff
@@ -154,5 +154,147 @@
   "usingXPathNotRecommended": "Using XPath locators is not recommended and can lead to fragile tests. Ask your development team to provide unique accessibility locators instead!",
   "Tap": "Tap",
   "Gathering initial app source…": "Gathering initial app source…",
-  "couldNotObtainSource": "Could not obtain source: {{errorMsg}}"
+  "couldNotObtainSource": "Could not obtain source: {{errorMsg}}",
+  "iOS": "iOS",
+  "Android": "Android",
+  "desiredCapabilitiesDocumentation": "Desired Capabilities Documentation",
+  "startSession": "Start Session",
+  "saveAs": "Save As...",
+  "attachToSession": "Attach to Session",
+  "Hello": "Hello",
+  "selectedElement": "Selected Element",
+  "back": "&lt;&lt; Back",
```

Comment:
I assume these must be escaped by i18next-react automatically, so you could put `<<` instead